tasks/main.yml

```diff
@@ -4,14 +4,23 @@
     update_cache: yes
   when: ansible_os_family == 'Debian'
 
-- name: install the required dependencies (Debian)
+- name: install the required dependencies (Debian) python3
+  apt:
+    name:
+    - openssl
+    - python3-boto
+    - python3-openssl
+    state: present
+  when: ansible_os_family == 'Debian' and ansible_python_version is version_compare('3', '>=')
```

Comment:
The documentation says that `version_compare` was renamed to `version` in 2.5:
https://docs.ansible.com/ansible/latest/user_guide/playbooks_tests.html#version-comparison

Could you please use `version` and verify that this works?

Reply:
Done